python_benchmarks/QRcodes/QR.py

```diff
@@ -0,0 +1,94 @@
+import glob
```

Comment:
Item 1 of OpenCV coding style guide: https://github.com/opencv/opencv/wiki/Coding_Style_Guide

> All the file names are written in lower case for better compatibility with both POSIX and Windows.

So,

- `qr_codes`
- `qr.py`
- `readme.md` or prefer `README.md` to highlight it (exception is possible as it is not used in other places)

Reply:
fixed

Reply:
Now qr.py and README.md